Samples/src/VisualDebugger/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Quantum.Simulation.Simulators;
+
+namespace vis_sim
+{
+    internal class Program
```

Comment:
Similarly, let's add some description about what this program does and how to call it from the command line.
A couple of troubleshooting tips would be nice (although those can go in the README.md)

Reply:
Did you run into any issues that you think should be mentioned in a troubleshooting section?